data/abi/abi_encode_test.go

```diff
@@ -100,6 +98,7 @@ func TestEncodeValid(t *testing.T) {
 	// encoding test for address, since address is 32 byte, it can be considered as 256 bit uint
 	// randomly generate 1000 uint256 and make address values, check if encoded value match with expected
 	upperLimit := big.NewInt(0).Lsh(big.NewInt(1), 256)
+	typeAddress := MakeAddressType()
```

Comment:
Perhaps the address type should/could be a constant, rather than function?  There's no utility to have multiple instances, right?